core/trino-server-rpm/src/test/java/io/trino/server/rpm/ServerIT.java

```diff
@@ -96,51 +138,6 @@ private static void assertPathDeleted(GenericContainer<?> container, String path
         assertEquals(actualResult.getExitCode(), 0);
     }
 
-    private static void testServer(String rpmHostPath, String expectedJavaVersion)
-    {
-        String rpm = "/" + new File(rpmHostPath).getName();
-
-        String command = "" +
-                // install RPM
-                "yum localinstall -q -y " + rpm + "\n" +
-                // update default JDK to 17
-                "alternatives --set java /usr/lib/jvm/zulu-17/bin/java\n" +
-                "alternatives --set javac /usr/lib/jvm/zulu-17/bin/javac\n" +
-                // create Hive catalog file
-                "mkdir /etc/trino/catalog\n" +
-                "echo CONFIG_ENV[HMS_PORT]=9083 >> /etc/trino/env.sh\n" +
-                "echo CONFIG_ENV[NODE_ID]=test-node-id-injected-via-env >> /etc/trino/env.sh\n" +
-                "sed -i \"s/^node.id=.*/node.id=\\${ENV:NODE_ID}/g\" /etc/trino/node.properties\n" +
-                "cat > /etc/trino/catalog/hive.properties <<\"EOT\"\n" +
-                "connector.name=hive\n" +
-                "hive.metastore.uri=thrift://localhost:${ENV:HMS_PORT}\n" +
-                "EOT\n" +
-                // create JMX catalog file
-                "cat > /etc/trino/catalog/jmx.properties <<\"EOT\"\n" +
-                "connector.name=jmx\n" +
-                "EOT\n" +
-                // start server
-                "/etc/init.d/trino start\n" +
-                // allow tail to work with Docker's non-local file system
-                "tail ---disable-inotify -F /var/log/trino/server.log\n";
-
-        try (GenericContainer<?> container = new GenericContainer<>(BASE_IMAGE)) {
-            container.withExposedPorts(8080)
-                    // the RPM is hundreds MB and file system bind is much more efficient
-                    .withFileSystemBind(rpmHostPath, rpm, BindMode.READ_ONLY)
-                    .withCommand("sh", "-xeuc", command)
-                    .waitingFor(forLogMessage(".*SERVER STARTED.*", 1).withStartupTimeout(Duration.ofMinutes(5)))
-                    .start();
-            QueryRunner queryRunner = new QueryRunner(container.getHost(), container.getMappedPort(8080));
-            assertEquals(queryRunner.execute("SHOW CATALOGS"), ImmutableSet.of(asList("system"), asList("hive"), asList("jmx")));
-            assertEquals(queryRunner.execute("SELECT node_id FROM system.runtime.nodes"), ImmutableSet.of(asList("test-node-id-injected-via-env")));
-            // TODO remove usage of assertEventually once https://github.com/trinodb/trino/issues/2214 is fixed
-            assertEventually(
-                    new io.airlift.units.Duration(1, MINUTES),
-                    () -> assertEquals(queryRunner.execute("SELECT specversion FROM jmx.current.\"java.lang:type=runtime\""), ImmutableSet.of(asList(expectedJavaVersion))));
-        }
```

Comment:
Any rason this looks like removed?

Reply:
it was inlined into calling method

Reply:
would it be possible to avoid this change, so that it's more apparent what are the material changes?

Reply:
I've extracted method inlining to a separate commit